### DISCUSSION TITLE
Trouble with execute script in Wotlk Classic

### DISCUSSION THREAD

**Discussion by DavidBennett604** [Backupiseasy/ThreatPlates#358] [Q&A]
I can't seem to make this work, I'm not a dev so I'm a bit out of my element but I imported the string correctly I think. I'm not sure if the script isn't working because it's for retail and not classic or if it's because I'm a hunter trying to use kill shot instead of a warrior using execute.  I'm using version 11.0.0-alpha3. Is there an easy fix for this? I tried googling solutions but it keeps pointing me back here.

**Comment by Backupiseasy** [comment]
Yes, it should be that you are a hunter. The script is only enabled for warriors. You can see that in the IsEnabled function:

```
function()
  -- Return false or nil, to disable the script, e.g., when it should be only active on a certain class
  local player_class = select(2, UnitClass("player"))
  return player_class == "WARRIOR"
end

```

To also make it work for hunters, change 
`return player_class == "WARRIOR" `
to 
`return player_class == "WARRIOR" or player_class == "HUNTER"`

I also try to update the script as well.

**Reply by DavidBennett604** [reply]
Hey this worked great! 2 follow ups here. First, in the appearance tab the scale setting doesn't seem to do anything, is there something else needed to make the bars scale 200% when below 20%? Second, to enable this execute script for all classes do I just delete the IsEnabled section?

**Reply by Backupiseasy** [reply]
I need to check that. By design, it should work that way, but scripting still has bugs. I also did look into execute for different classes, there it gets a lot more complicated :}
